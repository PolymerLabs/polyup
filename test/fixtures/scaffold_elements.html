<link rel='import' href='components/core-drawer-panel/core-drawer-panel.html'>
<link rel='import' href='components/core-header-panel/core-header-panel.html'>
<link rel='import' href='components/core-toolbar/core-toolbar.html'>
<link rel='import' href='components/core-menu/core-menu.html'>
<link rel='import' href='components/core-item/core-item.html'>
<link rel='import' href='components/paper-icon-button/paper-icon-button.html'>
<link rel='import' href='components/core-icons/core-icons.html'>

<core-drawer-panel id='drawerPanel'>
  <core-header-panel drawer>
    <core-toolbar id="navheader">
      <span>Menu</span>
    </core-toolbar>
    <core-menu>
      <core-item label="One"></core-item>
      <core-item label="Two"></core-item>
    </core-menu>
  </core-header-panel>

  <core-header-panel main>
    <core-toolbar id="mainheader">
      <paper-icon-button
        id="navicon" icon="menu"></paper-icon-button>
      <span flex>Title</span>
    </core-toolbar>
    <div class="content">
      If drawer is hidden, press button to display drawer.
    </div>
  </core-header-panel>
</core-drawer-panel>

<script>
  document.querySelector('#navicon').addEventListener('click', function() {
    document.querySelector('#drawerPanel').togglePanel();
  });
</script>
