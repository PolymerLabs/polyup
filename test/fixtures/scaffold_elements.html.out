<link rel="import" href="components/paper-drawer-panel/paper-drawer-panel.html">
<link rel="import" href="components/paper-header-panel/paper-header-panel.html">
<link rel="import" href="components/paper-toolbar/paper-toolbar.html">
<link rel="import" href="components/paper-menu/paper-menu.html">
<link rel="import" href="components/paper-item/paper-item.html">
<link rel="import" href="components/paper-icon-button/paper-icon-button.html">
<link rel="import" href="components/iron-icons/iron-icons.html">

<paper-drawer-panel id="drawerPanel">
  <paper-header-panel drawer="">
    <paper-toolbar id="navheader">
      <span>Menu</span>
    </paper-toolbar>
    <paper-menu>
      <paper-item undefined="One"></paper-item>
      <paper-item undefined="Two"></paper-item>
    </paper-menu>
  </paper-header-panel>

  <paper-header-panel main="">
    <paper-toolbar id="mainheader">
      <paper-icon-button id="navicon" icon="menu"></paper-icon-button>
      <span flex="">Title</span>
    </paper-toolbar>
    <div class="content">
      If drawer is hidden, press button to display drawer.
    </div>
  </paper-header-panel>
</paper-drawer-panel>

<script>
  document.querySelector('#navicon').addEventListener('click', function() {
    document.querySelector('#drawerPanel').togglePanel();
  });
</script>
